test/jdk/javax/swing/JColorChooser/Test4759306.java

```diff
@@ -21,22 +21,37 @@
  * questions.
  */
 
+import javax.swing.JColorChooser;
+import javax.swing.JFrame;
+import javax.swing.JPanel;
+
 /*
  * @test
  * @bug 4759306
+ * @library /test/jdk/java/awt/regtesthelpers /test/lib
+ * @build PassFailJFrame
  * @summary Checks if JColorChooser.setPreviewPanel removes the old one
- * @author Konstantin Eremin
-   @run applet/manual=yesno Test4759306.html
+ * @run main/manual Test4759306
  */
+public class Test4759306 {
+    public static void main(String[] args) throws Exception {
+        PassFailJFrame.builder()
+                .title("Test4759306")
+                .instructions("Check that there is no preview panel in the JColorChooser.")
+                .rows(5)
+                .columns(40)
+                .testTimeOut(10)
+                .testUI(Test4759306::test)
```

Comment:
Actually, we don't need to create a separate window for this, we can place the color chooser in the intruction window by using something like 
`.splitUIRight(Test4759306::createColorChooser)`

```java
private static JColorChooser createColorChooser() {
    JColorChooser chooser = new JColorChooser();
    chooser.setPreviewPanel(new JPanel());
    return chooser;
}
```